### PR TITLE
Fix icon button size

### PR DESCRIPTION
**Pull request by hadasfa**
צילום מסך 2022-08-14 ב-13 45 03

### CODE REVIEW

File: src/components/Button/Button.scss [hunk 1]
```diff
@@ -73,11 +73,11 @@
     }
   }
   .monday-style-button--left-icon {
-    padding-right: $spacing-small;
+    margin-right: var(--spacing-small);
```
Comment (hadasfa) [inline src/components/Button/Button.scss:76]: because padding effect the actual size of the button icon (and then "24px" is not really 24px)

**Review by SergeyRoyt — APPROVED** [review]

File: src/components/Button/Button.jsx [hunk 2]
```diff
@@ -217,17 +217,17 @@ const Button = forwardRef(
 
     const leftIconSize = useMemo(() => {
       if (typeof leftIcon !== "function") return;
-      return "24";
+      return "20";
```
Comment (sahariko) [inline src/components/Button/Button.jsx:220]: I see this number and a couple more being used in multiple components, why not add this to a constant?

File: src/components/IconButton/IconButton.jsx [hunk 3]
```diff
@@ -44,9 +44,9 @@ const IconButton = forwardRef(
     const iconSize = useMemo(() => {
       if (size === IconButton.sizes.XXS) return 16;
       if (size === IconButton.sizes.XS) return 16;
-      if (size === IconButton.sizes.SMALL) return 24;
-      if (size === IconButton.sizes.MEDIUM) return 24;
-      if (size === IconButton.sizes.LARGE) return 24;
+      if (size === IconButton.sizes.SMALL) return 20;
+      if (size === IconButton.sizes.MEDIUM) return 20;
+      if (size === IconButton.sizes.LARGE) return 20;
       return 24;
```
Comment (sahariko) [inline src/components/IconButton/IconButton.jsx:50]: Consider using a switch-case:
```js
switch (size) {
  case IconButton.sizes.XXS:
  case IconButton.sizes.XS:
    return 16;
  case IconButton.sizes.SMALL:
  case IconButton.sizes.MEDIUM:
  case IconButton.sizes.LARGE:
    return 20;
  default:
    return 24;
}
```

Comment (hadasfa) [reply src/components/IconButton/IconButton.jsx:50]: it should be just a quick fix but I guess we can refactor it a bit

**Pull request merged**